CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 0.20.1
+
+### Changed
+
```

Comment:
nit Usually we don't have a linebreak between header titles and the first bullet point.

Reply:
When I was working on Azure docs, the checking tool recommends adding a blank line before list of items. As far as I know, it's a more generally accepted markdown rule.